src/components/Page/base.css

```diff
@@ -29,6 +29,7 @@ html {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   text-rendering: optimizeLegibility;
+  scroll-behavior: smooth;
```

Comment:
@rogermparent this affects other pages, including docs. We didn't want this behavior on docs for a good reason (it takes too long to scroll if page is long and open a link someone shared with you, or click on the right side bar).

Reply:
Ah, okay! I'll revert this line and we can go for another approach.